src/libPersistence/BlockStorage.cpp

```diff
@@ -536,6 +536,7 @@ bool BlockStorage::GetAllTxBlocks(std::list<TxBlockSharedPtr>& blocks) {
 
   leveldb::Iterator* it =
       m_txBlockchainDB->GetDB()->NewIterator(leveldb::ReadOptions());
+  uint64_t count = 0;
```

Comment:
uint64_t seems an overkill

Reply:
number of TxBlks?

Reply:
it is just a counter right, uint32_t should be enough, unless u wanna count more than 4294967295

Reply:
yes we may need it...
we use uint64_t for all epoch, blocknums..